### PR TITLE
Update basic.md

### PR DESCRIPTION
**Pull request by AstrakhantsevaAA**
fix links to response actions page

### CODE REVIEW

File: docs/website/docs/dlt-ecosystem/verified-sources/rest_api/basic.md [hunk 1]
```diff
@@ -341,7 +341,7 @@ The fields in the endpoint configuration are:
 - `json`: The JSON payload to be sent with the request (for POST and PUT requests).
 - `paginator`: Pagination configuration for the endpoint. See the [pagination](#pagination) section for more details.
 - `data_selector`: A JSONPath to select the data from the response. See the [data selection](#data-selection) section for more details.
-- `response_actions`: A list of actions that define how to process the response data. See the [response actions](#response-actions) section for more details.
+- `response_actions`: A list of actions that define how to process the response data. See the [response actions](../advanced#response-actions) section for more details.
```
Comment (AstrakhantsevaAA) [inline docs/website/docs/dlt-ecosystem/verified-sources/rest_api/basic.md:344]:
```suggestion
- `response_actions`: A list of actions that define how to process the response data. See the [response actions](advanced#response-actions) section for more details.
```

File: docs/website/docs/dlt-ecosystem/verified-sources/rest_api/basic.md [hunk 2]
```diff
@@ -877,7 +877,7 @@ See the [troubleshooting guide](../../../general-usage/incremental-loading.md#tr
 
 #### Getting HTTP 404 errors
 
-Some API may return 404 errors for resources that do not exist or have no data. Manage these responses by configuring the `ignore` action in [response actions](#response-actions).
+Some API may return 404 errors for resources that do not exist or have no data. Manage these responses by configuring the `ignore` action in [response actions](../advanced#response-actions).
```
Comment (AstrakhantsevaAA) [inline docs/website/docs/dlt-ecosystem/verified-sources/rest_api/basic.md:880]:
```suggestion
Some API may return 404 errors for resources that do not exist or have no data. Manage these responses by configuring the `ignore` action in [response actions](advanced#response-actions).
```

File: docs/website/docs/dlt-ecosystem/verified-sources/rest_api/basic.md [hunk 3]
```diff
@@ -341,7 +341,7 @@ The fields in the endpoint configuration are:
 - `json`: The JSON payload to be sent with the request (for POST and PUT requests).
 - `paginator`: Pagination configuration for the endpoint. See the [pagination](#pagination) section for more details.
 - `data_selector`: A JSONPath to select the data from the response. See the [data selection](#data-selection) section for more details.
-- `response_actions`: A list of actions that define how to process the response data. See the [response actions](#response-actions) section for more details.
+- `response_actions`: A list of actions that define how to process the response data. See the [response actions](advanced#response-actions) section for more details.
```
Comment (AstrakhantsevaAA) [inline docs/website/docs/dlt-ecosystem/verified-sources/rest_api/basic.md:344]:
```suggestion
- `response_actions`: A list of actions that define how to process the response data. See the [response actions](./advanced#response-actions) section for more details.
```

File: docs/website/docs/dlt-ecosystem/verified-sources/rest_api/basic.md [hunk 4]
```diff
@@ -877,7 +877,7 @@ See the [troubleshooting guide](../../../general-usage/incremental-loading.md#tr
 
 #### Getting HTTP 404 errors
 
-Some API may return 404 errors for resources that do not exist or have no data. Manage these responses by configuring the `ignore` action in [response actions](#response-actions).
+Some API may return 404 errors for resources that do not exist or have no data. Manage these responses by configuring the `ignore` action in [response actions](advanced#response-actions).
```
Comment (AstrakhantsevaAA) [inline docs/website/docs/dlt-ecosystem/verified-sources/rest_api/basic.md:880]:
```suggestion
Some API may return 404 errors for resources that do not exist or have no data. Manage these responses by configuring the `ignore` action in [response actions](./advanced#response-actions).
```

**Review by burnash — APPROVED** [review]
Thanks @AstrakhantsevaAA!

**Pull request merged**